librz/util/print.c

```diff
@@ -1194,13 +1194,11 @@ RZ_API void rz_print_hexdump(RzPrint *p, ut64 addr, const ut8 *buf, int len, int
 					if (comment) {
 						if (p && p->colorfor) {
 							a = p->colorfor(p->user, addr + j, true);
-							if (a && *a) {
-								b = Color_RESET;
-							} else {
-								a = b = "";
+							if (!a || !*a) {
```

Comment:
are you sure `b` is dead here? This is inside a loop and `b` is used up in the code.

Reply:
b is used line 1026, 1028, 1030

defined line 1003, 1005, 1008

So everytime b is used it has been defined before (1003, 1005 or 100

Reply:
Good point, thanks for double checking ;)